submissions/adekemiAwomolo/PhpAndDtabase/customersView.php

```diff
@@ -0,0 +1,67 @@
+<?php
+require_once('connection.php');
+
+    try{
+    global $pdo;
+    $sql = "SELECT  orders.unit_price, orders.quantity, orders.total_price,ordered_time.created_at,ordered_time.id, 
+    product_details.name,customers_address.street,customers_address.city,customers_address.state,customers_address.country
+                    FROM orders
+                     LEFT JOIN ordered_time ON ordered_time.id = orders.id
+                    LEFT JOIN product_details ON product_details.id = orders.id
+                    LEFT JOIN customers_address ON customers_address.id = orders.id
+                ";
```

Comment:
What you here (https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/blob/master/submissions/adekemiAwomolo/database/init.sql) as your table names are different from your query. Study your tables and know-how relationships exist between these tables.

E.g There is a relationship using `single_orders` and `orders` table using the `order_id` column,
There is a relationship using `orders` and `products_details` table using the `product_id` column,
There is a relationship between `orders` and `customers` table using the `customer_id` column,
There is a relationship between `customers_information` and `customers` table `customers_id`. 

So, it is on these tables I was expecting to see in the `joins`.

Reply:
I did all that ma.Am I to change the id names , I gave the name “Id” to all my tables , am I suppose to give my the id’s for all my tables a different name @adaugochi

Reply:
Good evening @adaugochi .Thank you very much , i was using a different database, i have corrected the mistakes.Kindly review ma. Regards.